include/revng/Yield/Internal/Function.h

```diff
@@ -0,0 +1,58 @@
+#pragma once
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include <limits>
+#include <string>
+
+#include "revng/ADT/SortedVector.h"
+#include "revng/Model/VerifyHelper.h"
+#include "revng/Support/MetaAddress.h"
+#include "revng/Support/MetaAddress/YAMLTraits.h"
+#include "revng/Yield/Internal/BasicBlock.h"
+
+/* TUPLE-TREE-YAML
+
+name: Function
+type: struct
+fields:
+  - name: Entry
+    type: MetaAddress
+  - name: Name
+    type: std::string
+
+  - name: BasicBlocks
```

Comment:
Would it make sense to
```suggestion
  - name: ControlFlowGraph
```
For coherence with EFA?

Reply:
[Done](https://github.com/Cvelth/revng/blob/snapshot/wip_cfg_in_svg/include/revng/Yield/Function.h#L24).